packages/web-config-server/src/apiV1/measureBuilders/checkMultiConditionsByOrgUnit.js

```diff
@@ -0,0 +1,91 @@
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { groupAnalyticsByOrgUnit } from './helpers';
+
+/**
+ * Example use case: If yes to SchCVD012 but no to SchCVD012a display 'Yes', if yes to SchCVD012 and yes to SchCVD012a display 'Yes, smart TV'
+ * Example orgUnitConditions config:
+ * orgUnitConditions: {
+    Yes: {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
+      SchCVD012a: {
+        value: 'No',
+        operator: '=',
+      },
+    },
+    'Yes, smart TV': {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
+      SchCVD012a: {
+        value: 'Yes',
+        operator: '=',
+      },
+    },
+    No: {
+      SchCVD012: {
+        value: 'No',
+        operator: '=',
+      },
+    },
+ */
+class CheckMultiConditionsByOrgUnit extends DataBuilder {
+  async build() {
+    const { dataElementCode: queryDataCode } = this.query;
+    const { orgUnitConditions, dataElementCodes: configDataCodes } = this.config;
+
+    const dataElementCodes = configDataCodes || [queryDataCode];
```

Comment:
I hope we'll be able to move the `mapOverlay.dataElementCode` column inside `mapOverlay.measureBuilderConfig`, and make things simpler! Till then, I guess the above is fine.